Sources/Plasma/NucleusLib/pnNetCommon/plNetGroup.h

```diff
@@ -66,7 +66,13 @@ class plNetGroupId
    plNetGroupId() : fFlags(0) {}
    plNetGroupId(const plLocation& id, const uint8_t flags) : fId(id), fFlags(flags) {  }
    plNetGroupId(const plLocation& id) : fId(id), fFlags(0) {  }
-   
+   plNetGroupId(const plNetGroupId& copy) = default;
+   plNetGroupId(plNetGroupId&& move)
+       : fId(move.fId), fFlags(move.fFlags), fDesc(std::move(move.fDesc))
+   {
+       move.fId.Invalidate();
+   }
```

Comment:
These can be `= default` as above...  Furthermore, if all copy and move constructors are `= default` and there is no destructor, than all of them can be omitted to automatically get the default implementations.

Reply:
clang-tidy indicated that without the inclusion of the move constructor and operator that a copy would be performed. 😕 I'll try defaulting them all.

Reply:
If there's a copy constructor or copy operator declared at all (even if it's `= default`), it will suppress the creation of an automatic move constructor and move assignment operator.  You'd need to delete all 4 declarations (or set them all `= default`) to get all 4 automatically created.

Reply:
clang-tidy is pretty aggressively complaining about copying if I don't explicitly `default` the move constructor, even with all four declarations removed. So I guess that's what we'll have to do (sigh).

Reply:
Hmm, that may mean that something else is suppressing the move (maybe a destructor?)...  Making them explicit is certainly fine though

Reply:
Ah! It must be the defaulted virtual constructor on `plStateChangeNotifier` 😄